dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/feature/FeatureScope.kt

```diff
@@ -47,11 +47,13 @@ interface FeatureScope {
      * @param callerClass  name of the class calling the performance measurement.
      * @param metric name of the metric that we want to measure.
      * @param samplingRate value between 0-100 for sampling the event.
+     * @param operationName the name of the operation being measured
      * @return a PerformanceMetric object that can later be used to send telemetry, or null if sampled out
      */
     fun startPerformanceMeasure(
```

Comment:
side note: I added some comments to the original PR https://github.com/DataDog/dd-sdk-android/pull/1940#pullrequestreview-2037499101, but they were too late. I think in the future we should refactor this a bit a remove this method from the `FeatureScope`.